optional-deps/src/index.rs

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use core::ops::{Bound, RangeBounds};
+use pubgrub::range::Range;
+use pubgrub::type_aliases::Map;
+use pubgrub::version::NumberVersion as Version;
+use std::collections::{BTreeMap, BTreeSet as Set};
+
+/// Each package is identified by its name.
+pub type Package = String;
```

Comment:
Maybe this should be called `PackageName` so that we don't have 2 `Package` types?

Reply:
Yes that would be better indeed!